naresh/src/main/java/com/lftechnology/phpjava/assignmenttwo/SalutationSplitter.java

```diff
@@ -1,21 +1,45 @@
 package com.lftechnology.phpjava.assignmenttwo;
 
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.Scanner;
+
 /**
  * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
  */
-public class SalutationSplitter {
-    public static String name = "";
-    public static String salutaion = "";
-
-    public static void main(String[] args) {
-        for (int i = 0; i < args.length; i++) {
-            if (i == 0) {
-                name = args[0];
-            } else {
-                salutaion = salutaion + " " + args[i];
+public class SalutationSplitter implements MenuReader {
+    public void salutationSplitter(String input) {
+        String name = "";
+        String[] nameAndSalutation = input.split(" ");
+        String salutaion = nameAndSalutation[0];
+        String space = "";
+        for (int i = 0; i < nameAndSalutation.length; i++) {
+            if (i > 0) {
+                name += space + nameAndSalutation[i];
+                space = " ";
             }
         }
-        System.out.println(name);
-        System.out.println(salutaion);
+        System.out.println("\n\nSalutation : " + salutaion);
+        System.out.println("Name : " + name);
+    }
+
+
+    /**
+     * read the user input for the action
+     */
+    @Override
+    public void scanner() {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter the name and salutation.");
+        String nameAndSalutation = scan.nextLine();
+        salutationSplitter(nameAndSalutation);
+        System.out.println("\n\n\nPlease press any key to return the previous menu or enter 0 to exit. \n");
+        String input = scan.nextLine();
+        if (input.equals("0")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "0" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1132)